ros2launch/ros2launch/api/api.py

```diff
@@ -36,6 +37,32 @@ def __init__(self, msg, paths):
         self.paths = paths
 
 
+class NoKeystoreProvidedError(Exception):
+    """Exception raised when a keystore is not provided."""
+
+    def __init__(self):
+        super().__init__(('--no-create-keystore was specified and '
+                          'no keystore was provided'))
+
+
+class NonexistantKeystoreError(Exception):
+    """Exception raised when keystore is not on disk."""
+
+    def __init__(self, keystore_path):
+        super().__init__(('--no-create-keystore was specified and '
+                          f'ROS_SECURITY_KEYSTORE="{keystore_path}" '
```

Comment:
Ditto above: I thought we were ignoring these variables when launching, now? Leaving them as an implementation detail?

Reply:
Ping on this.

Reply:
c40721f